pnpr/docker/README.md

```
@@ -0,0 +1,73 @@
+# pnpr Docker image
+
+Official image for [`pnpr`](../), the pnpm-compatible npm registry server,
+published to GitHub Container Registry.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)